survey/views/survey_detail.py

```diff
@@ -36,6 +43,12 @@ def post(self, request, *args, **kwargs):
         form = ResponseForm(
             request.POST, survey=survey, user=request.user, step=kwargs.get("step", 0)
         )
+        if not survey.editable_answers and form.response is not None:
+            LOGGER.info(
+                """Redirects to survey list after trying
```

Comment:
This can probably fit on one line with standard quotes?

Reply:
Fixed with 7288fb9cb79f120f6c8b0da6fef0c47be8c47396